pkg/controller/broker/v2alpha4/activemqartemis/upgrade.go

```diff
@@ -0,0 +1,96 @@
+package v2alpha4activemqartemis
+
+import (
+	"fmt"
+	api "github.com/artemiscloud/activemq-artemis-operator/pkg/apis/broker/v2alpha4"
+	"github.com/artemiscloud/activemq-artemis-operator/version"
+	"strings"
+)
+
+const (
+	// LatestVersion product version supported
+	LatestVersion        = "0.2.0"
+	CompactLatestVersion = "020"
+	// LastMicroVersion product version supported
+	LastMicroVersion = "0.2.0"
+	// LastMinorVersion product version supported
+	LastMinorVersion = "0.1.0"
+)
+
+// SupportedVersions - product versions this operator supports
+var SupportedVersions = []string{LatestVersion, LastMicroVersion, LastMinorVersion}
+var OperandVersionFromOperatorVersion map[string]string = map[string]string{
+	"0.16.0": "0.1.0",
+	"0.17.0": "0.2.0",
```

Comment:
@gaohoward What does 0.18.0 support?

Reply:
I think we should support 0.2.0 as 0.17.0 does.